src/secp_ops.rs

```diff
@@ -7,8 +7,8 @@ use k256::ecdsa::{Signature as K1Signature, VerifyingKey as K1VerifyingKey};
 use p256::ecdsa::signature::hazmat::PrehashVerifier;
 use p256::ecdsa::{Signature as P1Signature, VerifyingKey as P1VerifyingKey};
 
-const SECP256R1_VERIFY_COST: Cost = 1850000;
-const SECP256K1_VERIFY_COST: Cost = 1300000;
+pub const SECP256R1_VERIFY_COST: Cost = 1850000;
```

Comment:
Prob don't need this change anymore.

Reply:
I wonder if all of the costs should be defined in one place, like `src/costs.rs` and exported publicly, for consistency?

Reply:
To avoid changing the public API on accident, I changed it to `pub(crate)` for now.